src/modules/params.rs

```diff
@@ -49,7 +47,7 @@ async fn prep_address(args_address: String, config: &Config) -> Result<String> {
 fn prep_unit(args_unit: String, config_unit: Option<&String>) -> Result<String> {
 	let unit = if args_unit.is_empty() && config_unit.is_some() {
 		match config_unit {
-			unit if unit == Some(&"°F".to_string()) => "fahrenheit",
```

Comment:
What we save in the config is the string from the unit in open-meteos json response which is either `°F` or `°C`. Changing this would make specifying fahrenheit as default value impossible.

Reply:
not true anymore. My memory got messed up here as I used it like that for the majority of the time prior releasing the app.